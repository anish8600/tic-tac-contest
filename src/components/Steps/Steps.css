.step{
    border-radius: 3em;
  }