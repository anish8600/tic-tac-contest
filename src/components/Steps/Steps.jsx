import React from 'react';

const Steps = () => <li></li>

export default Steps;