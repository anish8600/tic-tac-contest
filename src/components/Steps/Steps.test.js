import React from 'react';
import { shallow } from 'enzyme';
import Steps from './Steps';

describe('Steps', () => {
  let wrapper;
  beforeEach(() => wrapper = shallow(<Steps chances={[jest.fn()]} onClick={jest.fn()} />));

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot()
  });
  
  it('should render a <li />', () => {
    expect(wrapper.find('button').length).toEqual(1);
  });
});