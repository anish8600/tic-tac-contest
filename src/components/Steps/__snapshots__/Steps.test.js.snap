// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Steps should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Steps
    chances={
        Array [
            [Function],
          ]
    }
    onClick={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": "0",
    "nodeType": "host",
    "props": Object {
      "children": <button
        className="step"
        onClick={[Function]}
>
        Reset Game
</button>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Reset Game",
        "className": "step",
        "onClick": [Function],
      },
      "ref": null,
      "rendered": "Reset Game",
      "type": "button",
    },
    "type": "li",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": "0",
      "nodeType": "host",
      "props": Object {
        "children": <button
          className="step"
          onClick={[Function]}
>
          Reset Game
</button>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Reset Game",
          "className": "step",
          "onClick": [Function],
        },
        "ref": null,
        "rendered": "Reset Game",
        "type": "button",
      },
      "type": "li",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
